src/services/cacheService.ts

```diff
@@ -105,17 +115,27 @@ class CacheService {
         config,
         contentType
       )
-      .then(response => response.text())
-      .then(text =>
-        text == ''
-          ? <ActionResponse>{
-              message: 'Cache ' + cacheName + ' created with success',
-              success: true
-            }
-          : <ActionResponse>{ message: text, success: false }
-      )
-      .catch(
-        error => <ActionResponse>{ message: error.toString(), success: false }
+      .then(response => {
+        if (response.ok) {
+          return response.text();
+        }
+        throw response;
+      })
+      .then(text => {
+        let sucessMsg =
+          text == '' ? 'Cache ' + cacheName + ' created with success' : text;
+        return <ActionResponse>{
+          message: sucessMsg,
```

Comment:
Perform the action inline?
` message: text == '' ? 'Cache ' + cacheName + ' created with success' : text,